packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -332,11 +333,14 @@ class DynamicSidecarServiceLabels(BaseModel):
         description="allow complete internet access to containers in here",
     )
 
-    callbacks_mapping: Json[CallbacksMapping] | None = Field(
-        default_factory=CallbacksMapping,  # type: ignore[arg-type] # this one ANE I am not sure about
-        alias="simcore.service.callbacks-mapping",
-        description="exposes callbacks from user services to the sidecar",
-    )
+    callbacks_mapping: Annotated[
+        Json[CallbacksMapping] | None,
+        Field(
+            default_factory=CallbacksMapping,  # NOTE: PC->ANE I still think this could be an issue
```

Comment:
why do you thing this could still be an issue?

Reply:
a bit long to write. better discuss offline